backend/cmd/server/repository/resources/graphs/auth_flow_config_basic.json

```diff
@@ -8,6 +8,16 @@
             "executor": {
                 "name": "BasicAuthExecutor"
             },
+            "next": [
```

Comment:
This means authorization will not be available by default. This has to be added for the flow to use RBAC support?